### PR TITLE
feat(chat-e2e): added 'gpt-4o-mini-2024-07-18' model to api tests

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

added 'gpt-4o-mini-2024-07-18' model to api tests

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

File: apps/chat-e2e/src/tests/chatApi/arithmeticRequest.test.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ for (const modelToUse of modelsForArithmeticRequest) {
       const conversation = conversationData.prepareModelConversation(
         0,
         modelToUse.isSysPromptAllowed
-          ? 'Answer arithmetic question. The answer should be number, do not use natural language'
+          ? 'Compute arithmetic expression. The answer should be a number, do not use natural language'
```
Comment (nartovm) [inline apps/chat-e2e/src/tests/chatApi/arithmeticRequest.test.ts]: It is better to use
"Evaluate the given arithmetic expression and return only the numerical result. Do not include any explanations, units, or additional text. Provide the answer in its simplest form, using scientific notation for very large or small numbers when appropriate. Support basic arithmetic operations (+, -, *, /), exponents (^), parentheses, and common mathematical functions (sqrt, sin, cos, tan, log, ln). Round the result to 6 decimal places if necessary."
image
image

First screenshot is your sys.prompt
Second - the one I suggest

Comment (irinakartun) [reply apps/chat-e2e/src/tests/chatApi/arithmeticRequest.test.ts]: done

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1877.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by nartovm — DISMISSED** [review]

**Review by nartovm — APPROVED** [review]

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1877.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1877.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review/deploy-review

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1877.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**